video_xblock/utils.py

```diff
@@ -93,3 +96,33 @@ def remove_escaping(text):
         "&lt;": "<"
     }
     return unescape(text, html_unescape_table)
+
+
+def create_reference(lang_label, video_id, source="default"):
+    """
+    Build transcript file reference based on input information.
```

Comment:
It looks like it should actually be `reference name` rather than just `reference`.